### PR TITLE
securityhub_standards_control: improve docs for control ARNs

### PR DESCRIPTION
**Pull request by ulfjack**
### Description

Improve docs to help users find the right control ARNs.

### CODE REVIEW

File: website/docs/r/securityhub_standards_control.markdown [hunk 1]
```diff
@@ -37,7 +37,7 @@ resource "aws_securityhub_standards_control" "ensure_iam_password_policy_prevent
 
 The following arguments are supported:
 
-* `standards_control_arn` - (Required) The standards control ARN.
+* `standards_control_arn` - (Required) The standards control ARN. See the AWS tool documentation for how to list existing controls, https://docs.aws.amazon.com/cli/latest/reference/securityhub/get-enabled-standards.html and https://docs.aws.amazon.com/cli/latest/reference/securityhub/describe-standards-controls.html.
```
Comment (justinretzolk) [inline website/docs/r/securityhub_standards_control.markdown:40]:
```suggestion
* `standards_control_arn` - (Required) The standards control ARN. See the AWS documentation for how to list existing controls using [`get-enabled-standards`](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/securityhub/get-enabled-standards.html) and [`describe-standards-controls`](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/securityhub/describe-standards-controls.html).
```

**Review by justinretzolk — CHANGES_REQUESTED** [review]
Thank you for taking the time to raise this! I made a couple of minor suggestions here that I think we should do prior to merging this:

- Updated the links to point to AWC CLI v2 documentation
- Updated to use formatted links rather than bare URLs

**Comment by ulfjack** [comment]
Thanks! Any other feedback?

**Review by justinretzolk — APPROVED** [review]
Looks good to me! Thank you again for taking the time to raise this! I'll get this merged once the checks finish up 🚀

**Pull request merged**